src/main/webapp/WEB-INF/templates/homepage.jsp

```diff
@@ -5,15 +5,11 @@
 <html>
   <head>
     <t:head/>
+    <link href="../searchbox-style.css" rel="stylesheet" type="text/css">
     <title>Project Homepage</title>
   </head>
   <body>
     <t:base/>
-    <p>Hello World!</p>
-    <p>The current date is ${requestScope["date"]}</p>
-    <form method="GET" action="/search">
-      <input name="q" placeholder="Query goes here" required>
-      <input type="submit">
-    </form>
+    <div class="gcse-searchbox-only" data-resultsUrl="/search"></div>
```

Comment:
The homepage placement of the search box is identical to the srp placement. Will this be changed to look closer actually Google homepage with a page centered searchbox?

Reply:
For the MVP, I think the unchanging searchbox placement is fine. The placement (along with the searchbox styling) can definitely be changed later to resemble the Google homepage.

Reply:
From the design doc I got the impression that there wasn't going to be a homepage anymore. Has that changed?

Also, the design doc's mock's searchbox doesn't extend across the entire page, but that's not super important to address now.